### ISSUE TITLE
main-figures/KeyParameterGelman_MultChain needs re-making

### ISSUE THREAD

**Issue by cgrandin** [cgrandin/hake-assessment#481]
Located in main-figures.rnw

**Comment by aaronmberger** [comment]
Yes, this will need to be done. I'll assign to myself and will do if we have time. This is a multi-chain convergence test so it requires >1 MCMC runs of the same length, thin rate, and burn-in - only difference is a different seed.  Last year we compared 2 runs, which is standard for large models.  I will set off another MCMC run with a different seed if the RData file that includes the first MCMC run builds today.  

This approach is different than the standard MCMC diagnostics of looking at different segments of the same chain.  We added the multi-chain (or two-chain) comparison last year in response to an SRG request so not a huge concern if not completed.  Nonetheless, not hard to do just takes CPU time.
